core/sqf/sqenvcom.sh

```diff
@@ -795,13 +799,9 @@ fi
 
 # Non-standard or newer version tools
 export BISON="${TOOLSDIR}/bison_3_linux/bin/bison"     # Need 1.3 version or later
-
-
 export LLVM="${TOOLSDIR}/dest-llvm-3.2"
 export UDIS86="${TOOLSDIR}/udis86-1.7.2"
 export ICU="${TOOLSDIR}/icu4.4"
-export QT_TOOLKIT="${TOOLSDIR}/Qt-4.8.5-64"
-
 
```

Comment:
That's fine, but Jenkins and most developers routinely build the GUI on our machines today, and that should continue to work. We use the DISPLAY command (invoking the GUI) all the time to debug queries.